sdk/core/core-http/lib/atomResourceSerializerBase.ts

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { Constants } from "./util/constants";
+import { serializeJsonToAtomXml } from "./util/xml";
+import { parseResultFromAtomResponse } from "./util/atomHandler";
+
+export class AtomResourceSerializerBase {
+  static serializeToAtomXmlRequest(
+    resourceName: string,
+    resource: any,
+    properties: string[],
+    xmlNamespace: string
+  ): string {
+    const content: any = {};
```

Comment:
Updated.

Also, I had checked with Amar, Neeraj on the XML schema / structure for the requests/responses and if it was known, it seems like there's no such thing being maintained as such. So we'll need to infer structure based on code and getting things to work.
As such, since implementation may change - so it doesn't seem all that valuable to get strict about representing the JSON data.